Info 2021-11-20 18:58:49: Experimentation service retrieved: [object Object]
Info 2021-11-20 18:58:49: ZMQ install verified.
Error 2021-11-20 18:58:50: Jupyter Extension (Error in Failed to update settings.json, method:fixSettingsFile): [EntryNotFound (FileSystemError): Unable to read file '/home/yli/.config/Code/User/settings.json' (Error: Unable to resolve nonexistent file '/home/yli/.config/Code/User/settings.json')
	at _handleError (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:94:164282)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
	at async m.readFile (/home/yli/.vscode/extensions/ms-toolsai.jupyter-2021.10.1101450599/out/client/extension.js:102:789683)
	at async g.fixSettingsFile (/home/yli/.vscode/extensions/ms-toolsai.jupyter-2021.10.1101450599/out/client/extension.js:66:322174)
	at async Promise.all (index 0)
	at async g.updateSettings (/home/yli/.vscode/extensions/ms-toolsai.jupyter-2021.10.1101450599/out/client/extension.js:66:323924)] {
  code: 'FileNotFound'
}
Error 2021-11-20 18:58:50: Jupyter Extension (Error in Failed to update settings.json, method:fixSettingsFile): [EntryNotFound (FileSystemError): Unable to read file '/home/yli/assignment_5/.vscode/settings.json' (Error: Unable to resolve nonexistent file '/home/yli/assignment_5/.vscode/settings.json')
	at _handleError (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:94:164282)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
	at async m.readFile (/home/yli/.vscode/extensions/ms-toolsai.jupyter-2021.10.1101450599/out/client/extension.js:102:789683)
	at async g.fixSettingsFile (/home/yli/.vscode/extensions/ms-toolsai.jupyter-2021.10.1101450599/out/client/extension.js:66:322174)
	at async Promise.all (index 1)
	at async g.updateSettings (/home/yli/.vscode/extensions/ms-toolsai.jupyter-2021.10.1101450599/out/client/extension.js:66:323924)] {
  code: 'FileNotFound'
}
Info 2021-11-20 18:58:50: Experiment status for python is {"enabled":true,"optInto":[],"optOutFrom":[]}
Info 2021-11-20 18:58:53: Process Execution: > /usr/bin/python3 -m pip list
> /usr/bin/python3 -m pip list
Info 2021-11-20 18:58:53: Process Execution: > /usr/bin/python3 -m pip list
> /usr/bin/python3 -m pip list
Info 2021-11-20 18:58:53: Process Execution: > /usr/bin/python3 -c "import jupyter"
> /usr/bin/python3 -c "import jupyter"
Info 2021-11-20 18:58:53: Process Execution: > /usr/bin/python3 -c "import jupyter"
> /usr/bin/python3 -c "import jupyter"
Info 2021-11-20 18:58:53: Process Execution: > /usr/bin/python3 -c "import notebook"
> /usr/bin/python3 -c "import notebook"
Info 2021-11-20 18:58:53: Process Execution: > /usr/bin/python3 -c "import notebook"
> /usr/bin/python3 -c "import notebook"
Info 2021-11-20 19:14:23: Shutting down notebooks for 26fdaa0a-03a9-4b0a-a23b-1f7cf30e5ed6
Info 2021-11-20 19:14:23: Disposing HostJupyterExecution 06240c9c-bb69-429e-a632-85593ee2079a
Info 2021-11-20 19:14:23: Disposing super HostJupyterExecution 06240c9c-bb69-429e-a632-85593ee2079a
Info 2021-11-20 19:14:23: Disposing HostJupyterExecution 06240c9c-bb69-429e-a632-85593ee2079a
Info 2021-11-20 19:14:23: Finished disposing HostJupyterExecution  06240c9c-bb69-429e-a632-85593ee2079a
Info 2021-11-20 19:14:23: Cleaning up server cache 06240c9c-bb69-429e-a632-85593ee2079a
Info 2021-11-20 19:14:23: Finished disposing HostJupyterExecution  06240c9c-bb69-429e-a632-85593ee2079a
